.github/workflows/upload-sbom.yaml

```diff
@@ -0,0 +1,24 @@
+name: Syft SBOM Creation
```

Comment:
The name should match the workflow filename.

I suggest:

```suggestion
# .github/workflows/sbom.yaml
name: SBOM
```

What do you think?

Reply:
Sure, let’s do it!